src/components/html/EcApplication.html

```diff
@@ -1,131 +1,197 @@
 <div class="ec-application">
-  <a-row :gutter="24">
-    <div v-if="isLoaded">
-      <a-col :md="24" :lg="7">
-        <a-card>
-          <div class="text-center">
-            <a-avatar v-if="icon" :src="icon" :size="170"  class="mb-3 img-fluid"/>
-            <a-avatar
-              v-else
-              :size="170"
-              :style="{ background: randomColors, fontSize: '4rem' }"
-              class="mb-3 img-fluid"
-            >
-              {{ title.charAt(0) }}
-            </a-avatar>
-
-            <a-card-meta
-              :title="title"
-              class="overflow-hidden"
-            >
-              <template slot="description">
-                {{ shortDescription }}
-              </template>
-            </a-card-meta>
-          </div>
+  <div 
+    class="row d-flex justify-content-around" 
+    v-if="isLoaded"
+  >
+    <div class="col-3 card bg-white border h-100">
+      <div class="text-center">
+        <img 
+          v-if="icon" 
+          :src="icon" 
+          class="mb-3" 
+        />
+        <div 
+          v-else  
+          :style="{ background: randomColors, fontSize: '4rem' }" 
+          class="mb-3 img-fluid"
+        >
+          {{ title.charAt(0) }}
+        </div>
 
-          <div class="text-center mb-1 mt-4">
-            <span>
-              <a-button
-                shape="round"
-                type="primary"
-                size="large"
-                @click="installApp"
-                v-if="!isInstalled"
-              >
-                <i class="fas fa-plus mr-2"></i>
-                {{ i19install }}
-              </a-button>
+        <div class="card-body p-0">
+          <h5>
+            {{ title }}
+          </h5>
+          <p class="text-muted text-center">
+            {{ shortDescription }}
+          </p>
+        </div>
+      </div>
 
-              <a-popconfirm
-                :title="i19doYouWantToDeleteAppQn"
-                @confirm="uninstallApp"
-                :okText="i19yes"
-                :cancelText="i19no"
-                v-if="isInstalled"
-              >
-                <a-button
-                  shape="round"
-                  type="danger"
-                  size="large"
-                >
-                  <i class="fas fa-trash mr-2"></i>
-                  {{ i19uninstall }}
-                </a-button>
-              </a-popconfirm>
-            </span>
+      <div class="text-center mb-1 mt-4">
+        <span>
+          <div 
+            class="btn btn-pink rounded-pill" 
+            type="primary" 
+            size="large" 
+            @click="installApp" 
+            v-if="!isInstalled"
+          >
+            <i class="fas fa-plus mr-2"></i>
+            {{ i19install }}
           </div>
 
-          <div class="text-center mb-1">
-            <p>{{ price }}</p>
+          <div v-else>
+            <!--@click="uninstallApp"-->
+            <button class="btn btn-pink rounded-pill" id="popover">
```

Comment:
```suggestion
            <button class="btn btn-danger" id="popover">
```

Acho que não precisa do `.rounded-pill`, e como é uma ação destrutiva tem que ser `.btn-danger`.

> Não costumamos usar variações por cor assim não, geralmente é primário, secundário, light ou contexto.